crates/ra_syntax/src/syntax_error.rs

```diff
@@ -1,209 +1,47 @@
-//! FIXME: write short doc here
+//! Module that defines `SyntaxError`.
```

Comment:
Would be better to explain why, rather than what:

This module defines `SyntaxError`, the type we use to represent errors from
* tokenization
* parsing
* tree validation

The errors are unstructured: just a message and a location.

Reply:
I moved this description to doc comment on top of `SynraxError` itself, otherwise it would be repetition